src/app/components/media/AudioMediaCard.js

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import { Player } from '@meedan/check-ui';
+import FormControl from '@material-ui/core/FormControl';
+import MenuItem from '@material-ui/core/MenuItem';
+import Select from '@material-ui/core/Select';
+import styled from 'styled-components';
+import { FormattedMessage } from 'react-intl';
+import AspectRatio from '../layout/AspectRatio';
+import { units } from '../../styles/js/shared';
+
+const StyledPlaybackRate = styled.div`
+  margin-top: ${units(2)};
+  position: absolute;
+  right: 0;
+`;
+
+const AudioMediaCard = (props) => {
+  const [playbackRate, setplaybackRate] = React.useState(1);
+
+  return (
+    <article className="video-media-card" style={{ position: 'relative' }}>
+      <AspectRatio>
+        <Player
+          url={props.audioPath}
+          className="video-media-player"
+          playbackRate={playbackRate}
+          onDuration={d => props.setPlayerState({ duration: d })}
+          onPause={() => props.setPlayerState({ playing: false, gaps: [] })}
+          onPlay={() => props.setPlayerState({ playing: true })}
+          onProgress={p => props.setPlayerState({ progress: p })}
+          onReady={props.onPlayerReady}
+          onTimeUpdate={t => props.setPlayerState({ time: t })}
+          playing={props.playing}
+          start={props.start}
+          end={props.end}
+          gaps={props.gaps}
+          scrubTo={props.scrubTo}
+          seekTo={props.seekTo}
+        />
+      </AspectRatio>
+      <StyledPlaybackRate>
+        <FormControl variant="outlined">
+          <Select
+            id="video-media-card__playback-rate"
```

Comment:
wrong class name

Reply:
Used MediaPlayerCard component for both video & audio